api/src/services/fields.ts

```diff
@@ -668,6 +668,8 @@ export class FieldsService {
 			column = table.dateTime(field.field, { useTz: false });
 		} else if (field.type === 'timestamp') {
 			column = table.timestamp(field.field, { useTz: true });
+		} else if (field.type === 'json') {
+			column = table.jsonb(field.field);
```

Comment:
`jsonb` !== `json`, so is this a breaking change? Do we need to include migrations to upgrade existing json to binary json?

Reply:
I do not consider this a breaking change because the existing fields dont need to get migrated, however newly created fields after the update will be created as `jsonb`. The json logic checks the type and falls back to using the `to_jsonb()` function for existing `json` fields. I do suspect this comes at a performance cost.